anesthetic/samples.py

```diff
@@ -24,7 +24,7 @@ class MCMCSamples(WeightedDataFrame):
     Example plotting commands include
         - ``mcmc.plot_1d(['paramA', 'paramB'])``
         - ``mcmc.plot_2d(['paramA', 'paramB'])``
-        - ``mcmc.plot_2d(['paramA', 'paramB'], ['paramC', 'paramD'])``
+        - ``mcmc.plot_2d([['paramA', 'paramB'], ['paramC', 'paramD'])]``
```

Comment:
Closing brackets wrong way round: should be ]) but is )]

Reply:
In theory there are test suites that can test docstring code, but for now we'll need to rely on eagle-eyed reviewers like yourself